htdocs/admin/pdf.php

```diff
@@ -371,6 +373,18 @@
 //}
 print '</td></tr>';
 
+//
+
+print '<tr class="oddeven"><td>'.$langs->trans("DOC_SHOW_CUSTOMER_CODE");
```

Comment:
Can you invert the option (the default value must be as actual, not set and showing the customer ref on document). And setting must hide.
So can you rename option 'DOC_SHOW_CUSTOMER_CODE' into 'MAIN_DOC_HIDE_CUSTOMER_CODE'

Reply:
It's ok for me